### PR TITLE
[@wescpy] make GAE Quickstart samples consistent

### PR DESCRIPTION
**Pull request by wescpy**
These samples should be nearly identical (except for `app.yaml`). They've been edited by different engineers over the years and have drifted slightly apart. This PR aims to put them in-sync again. A pair of bugs were filed (within Google), and this PR will be linked to them as the fix.

### CODE REVIEW

File: appengine/flexible/hello_world/main.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 Google Inc. All Rights Reserved.
+# Copyright 2021 Google LLC
```
Comment (dandhlee) [inline appengine/flexible/hello_world/main.py:1]: Please leave the copyright year as-is!

File: appengine/standard_python3/hello_world/main.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2021 Google LLC
```
Comment (dandhlee) [inline appengine/standard_python3/hello_world/main.py:1]: Ditto.

**Review by dandhlee — CHANGES_REQUESTED** [review]

**Comment by wescpy** [comment]
reverted copyright years

File: appengine/flexible/hello_world/main.py (hunk 1 above)
Comment (wescpy) [reply appengine/flexible/hello_world/main.py]: DONE

File: appengine/standard_python3/hello_world/main.py (hunk 2 above)
Comment (wescpy) [reply appengine/standard_python3/hello_world/main.py]: DONE

**Review by dandhlee — APPROVED** [review]
LGTM.

**Review by leahecole — APPROVED** [review]

**Comment by wescpy** [comment]
thx all!

**Pull request merged**